shell/common/api/features.cc

```diff
@@ -66,6 +70,7 @@ void Initialize(v8::Local<v8::Object> exports,
                 v8::Local<v8::Context> context,
                 void* priv) {
   gin_helper::Dictionary dict(context->GetIsolate(), exports);
+  dict.SetMethod("isBuiltinSpellcheckerEnabled", &IsBuiltinSpellcheckerEnabled);
```

Comment:
_(nit)_ in the rest of the public API, it looks like we capitalize the first 'c' in 'spellcheck', and 'spellchecker' , e.g. "setSpellCheckProvider" and "listWordsInSpellCheckerDictionary"... we should use the same capitalization here for consistency

Reply:
Done.